flake.nix

```diff
@@ -128,5 +128,7 @@
         };
 
         formatter = pkgs.nixpkgs-fmt;
+
```

Comment:
`packages.default` is not really self-explanatory so I'd like to add the following comment above this line:
```
# `nix build` builds the packages.default output
```

Reply:
I thought since it's standard in flakes it wouldn't need a description. I'll add the comment though ;)